src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarSignal.tsx

```diff
@@ -0,0 +1,101 @@
+import { Flex, Text, Stack } from "@artsy/palette"
+import { useFragment, graphql } from "react-relay"
+import { ArtworkSidebarSignal_artwork$key } from "__generated__/ArtworkSidebarSignal_artwork.graphql"
+import FairIcon from "@artsy/icons/FairIcon"
+import { RouterLink } from "System/Components/RouterLink"
+import TrendingIcon from "@artsy/icons/TrendingIcon"
+import VerifiedIcon from "@artsy/icons/VerifiedIcon"
+
+interface ArtworkSidebarSignalProps {
+  artwork: ArtworkSidebarSignal_artwork$key
+}
+
+export const ArtworkSidebarSignal: React.FC<ArtworkSidebarSignalProps> = ({
+  artwork,
+}) => {
+  const data = useFragment(artworkSidebarSignalFragment, artwork)
+
+  const isShowingNow = data.collectorSignals?.runningShow ?? false
+  const shouldRenderCuratorsPick =
+    data.collectorSignals?.primaryLabel === "CURATORS_PICK"
+  const shouldRenderIncreasedInterest =
+    data.collectorSignals?.primaryLabel === "INCREASED_INTEREST"
```

Comment:
We might be able to use a switch statement or conditional here. What it if was just
```tsx
const primaryLabel = data.collectorSignals?.primaryLabel
if (primaryLabel === "CURATORS_PICK") {
  return (<Flex alignItems="top" my={4} data-testid="curators_pick">
    <VerifiedIcon color="black100" width={18} mr={1} mt={0.5} />
}
```
etc. A `switch` would be another option, but the need to account for showing now (should this go in `primaryLabel`?) might make that a little uglier.